src/pages/Account.vue

```diff
@@ -53,7 +58,10 @@
 
                 </div>
 
-                <button id="requestVoucherHistoryEmail" v-on:click="onRequestVoucherHistoryEmail">Email payment history</button>
+                <div class="cta-buttons">
+                    <button id="requestVoucherHistoryEmail" v-on:click="onRequestVoucherHistoryEmail">Email all payment history</button>
+                    <button v-on:click="onRequestSubmissionEmail" :disabled="selected">Email selected payment history</button>
```

Comment:
Kat was on board with the two buttons. The button on the right for selected history is greyed out and disabled until a radio button is selected though, which I think helps.